core/dbt/adapters/factory.py

```diff
@@ -1,23 +1,17 @@
 import threading
-from pathlib import Path
+from contextlib import contextmanager
```

Comment:
This is the only new import... all of the rest of the changes are from using isort to correctly sort the imports which wasn't done before... #leaveitbetter